src/Microsoft.Diagnostics.Monitoring.EventPipe/Configuration/AggregateSourceConfiguration.cs

```diff
@@ -28,5 +28,37 @@ public override bool RequestRundown
             get => _configurations.Any(c => c.RequestRundown);
             set => throw new NotSupportedException();
         }
+
+        public override long RundownKeyword
+        {
+            get => _configurations.Select(c => c.RundownKeyword).Aggregate((x, y) => x | y);
+            set => throw new NotSupportedException();
+        }
+
+        public override RetryStrategy RetryStrategy
+        {
+            get
+            {
+                RetryStrategy result = RetryStrategy.DoNotRetry;
+                foreach (MonitoringSourceConfiguration configurations in _configurations)
+                {
+                    if (result == RetryStrategy.DoNotRetry)
+                    {
+                        // Anything overrides DoNotRetry
```

Comment:
I don't think we should ever override DoNotRetry. I interpret DoNotRetry to mean a profile cared about getting exactly the keywords it asked for and if that can't be satisfied then we should not do the trace and report an error to the user.